src/app/components/Hits/Hits.jsx

```diff
@@ -100,7 +123,10 @@ class Hits extends React.Component {
     return (
       <div id="results-description" className="nypl-results-summary">
```

Comment:
seems like there are some `aria` attributes missing here… [see the toolkit](https://nypl.github.io/design-toolkit/sections/tables.html#summary)